docs/user/interop.rst

```diff
@@ -122,6 +123,7 @@ Operation        C syntax                 Scala Syntax
 Load value       ``*ptr``                 ``!ptr``
 Store value      ``*ptr = value``         ``!ptr = value``
 Pointer to index ``ptr + i``, ``&ptr[i]`` ``ptr + i``
+Pointer offset   ``ptr1 - ptr2``          ``ptr1 - ptr2``
```

Comment:
`Pointer offset` doesn't seem to be very clear. Lets try `Elements between` instead.